.travis.yml

```diff
@@ -2,6 +2,7 @@ language: cpp
 
 # OS config, depends on actual 'os' in build matrix
 dist: xenial
+sudo: true
```

Comment:
`sudo: true` is deprecated since December 2018, see https://changelog.travis-ci.com/the-container-based-build-environment-is-fully-deprecated-84517

Reply:
Thanks! Removed this :)